src/kex_sidh_msr/kex_sidh_msr.c

```diff
@@ -302,9 +302,11 @@ OQS_STATUS OQS_KEX_sidh_msr_alice_1(OQS_KEX *k, const void *alice_priv, const ui
 	return ret;
 }
 
-void OQS_KEX_sidh_msr_alice_priv_free(UNUSED OQS_KEX *k, void *alice_priv) {
+void OQS_KEX_sidh_msr_alice_priv_free(OQS_KEX *k, void *alice_priv) {
 	if (alice_priv) {
-		free(alice_priv);
+		// free(alice_priv);
+		SIDH_CTX *sidh_ctx = (SIDH_CTX *) k->ctx;
+		OQS_MEM_secure_free(alice_priv, (sidh_ctx->priv_key_len) * sizeof(uint8_t));
```

Comment:
I've noticed that I used free() in the err: label to free keys in case of error. That too should use OQS_MEM_secure_free as well. This could be done in this PR or in a later one...

Reply:
Do we need that actually? If it errors should we care about releasing safely the memory?

Reply:
I will remove the //free, kept it just to see it in code review